src/main/java/dev/walshy/sfmobdrops/SfMobDrops.java

```diff
@@ -84,6 +84,9 @@ protected void loadConfig() {
 
     @EventHandler
     public void onMobDeath(@Nonnull EntityDeathEvent e) {
+
+        if (getConfig().getBoolean("settings.only-player-drops", false) && e.getEntity().getKiller() == null) return;
```

Comment:
This path doesn't even exist in the config.

You did not test this code...

Reply:
whoops